src/SCREENS.ts

```diff
@@ -30,14 +49,162 @@ const SCREENS = {
             ADDRESS: 'Settings_Card_Get_Physical_Address',
             CONFIRM: 'Settings_Card_Get_Physical_Confirm',
         },
+        WALLET_TRANSFER_BALANCE: 'Settings_Wallet_Transfer_Balance',
+        WALLET_CHOOSE_TRANSFER_ACCOUNT: 'Settings_Wallet_Choose_Transfer_Account',
+        WALLET_ENABLE_PAYMENTS: 'Settings_Wallet_EnablePayments',
+        WALLET_CARD_ACTIVATE: 'Settings_Wallet_Card_Activate',
+        WALLET_REPORT_VIRTUAL_CARD_FRAUD: 'Settings_Wallet_ReportVirtualCardFraud',
+        WALLET_CARDS_DIGITAL_DETAILS_UPDATE_ADDRESS: 'Settings_Wallet_Cards_Digital_Details_Update_Address',
+
+        ADD_DEBIT_CARD: 'Settings_Add_Debit_Card',
+        ADD_BANK_ACCOUNT: 'Settings_Add_Bank_Account',
+        PREFERENCES_PRIORITY_MODE: 'Settings_Preferences_PriorityMode',
+        PREFERENCES_LANGUAGE: 'Settings_Preferences_Language',
+        PREFERENCES_THEME: 'Settings_Preferences_Theme',
+        CLOSE: 'Settings_Close',
+        STATUS_SET: 'Settings_Status_Set',
+        TWO_FACTOR_AUTH: 'Settings_TwoFactorAuth',
+        REPORT_CARD_LOST_OR_DAMAGED: 'Settings_ReportCardLostOrDamaged',
     },
     SAVE_THE_WORLD: {
         ROOT: 'SaveTheWorld_Root',
     },
+    RIGHT_MODAL: {
+        SETTINGS: 'Settings',
+        NEW_CHAT: 'NewChat',
+        SEARCH: 'Search',
+        DETAILS: 'Details',
+        PROFILE: 'Profile',
+        REPORT_DETAILS: 'Report_Details',
+        REPORT_SETTINGS: 'Report_Settings',
+        REPORT_WELCOME_MESSAGE: 'Report_WelcomeMessage',
+        PARTICIPANTS: 'Participants',
+        MONEY_REQUEST: 'MoneyRequest',
+        NEW_TASK: 'NewTask',
+        TEACHERS_UNITE: 'TeachersUnite',
+        TASK_DETAILS: 'Task_Details',
+        ENABLE_PAYMENTS: 'EnablePayments',
+        SPLIT_DETAILS: 'SplitDetails',
+        ADD_PERSONAL_BANK_ACCOUNT: 'AddPersonalBankAccount',
+        WALLET_STATEMENT: 'Wallet_Statement',
+        FLAG_COMMENT: 'Flag_Comment',
+        EDIT_REQUEST: 'EditRequest',
+        SIGN_IN: 'SignIn',
+        PRIVATE_NOTES: 'Private_Notes',
+        ROOM_MEMBERS: 'RoomMembers',
+        ROOM_INVITE: 'RoomInvite',
+        REFERRAL: 'Referral',
+    },
     SIGN_IN_WITH_APPLE_DESKTOP: 'AppleSignInDesktop',
     SIGN_IN_WITH_GOOGLE_DESKTOP: 'GoogleSignInDesktop',
     DESKTOP_SIGN_IN_REDIRECT: 'DesktopSignInRedirect',
     SAML_SIGN_IN: 'SAMLSignIn',
+
+    MONEY_REQUEST: {
+        ROOT: 'Money_Request',
+        AMOUNT: 'Money_Request_Amount',
```

Comment:
Should these be solid?

```suggestion
        ROOT: 'MoneRequest',
        AMOUNT: 'MoneyRequest_Amount',
```